src/Renderer/Hal.php

```diff
@@ -105,9 +105,9 @@ public function render($cache = false, $params = array())
 		$this->app->setHeader('Expires', '0', true);
 		$this->app->setHeader('Cache-Control', 'must-revalidate, post-check=0, pre-check=0', true);
 		$this->app->setHeader('Cache-Control', 'private', false);
-		$this->app->setHeader('Content-Type', $this->getMimeEncoding() . '; charset=' . $this->getCharset(), true);
-		$this->app->setHeader('Webservice-Name', $this->webservice->webserviceName, true);
-		$this->app->setHeader('Webservice-Version', $this->webservice->webserviceVersion, true);
+		$this->app->setHeader('Content-type', $this->getMimeEncoding() . '; charset=' . $this->getCharset(), true);
+		$this->app->setHeader('Webservice name', $this->webservice->webserviceName, true);
+		$this->app->setHeader('Webservice version', $this->webservice->webserviceVersion, true);
```

Comment:
Can you update your code?  You are reverting some fixes made a while back.  Thanks.